src/css/editable.less

```diff
@@ -5,7 +5,6 @@
     width: 100%;
     padding: 2px;
     .box-sizing(border-box); // supported by IE8+: http://caniuse.com/css3-boxsizing
-    white-space: pre;
```

Comment:
does this definitely belong just in mq-editable-field? what happens for statically rendered math?